java/piranha/src/main/java/com/uber/piranha/PiranhaRuntimeException.java

```diff
@@ -0,0 +1,12 @@
+package com.uber.piranha;
```

Comment:
As silly as that might be for this tiny exception class... can we get a copyright header here?

Reply:
Oops! This is what I get for copying and pasting from `PiranhaConfigurationException` 😂

Reply:
😅  yup, that one is on us... I need to go over the code and fix those java files that are still missing the header.